cd/utils/test_artifact_repository.py

```diff
@@ -178,11 +178,11 @@ def test_probe_variant_cpu(self, mock_features):
     @patch('artifact_repository.get_cuda_version')
     def test_probe_variant_cuda(self, mock_cuda_version, mock_features):
         """
-        Tests 'cu102' is returned if MKLDNN is OFF and CUDA is ON and CUDA version is 10.2
+        Tests 'cu100' is returned if MKLDNN is OFF and CUDA is ON and CUDA version is 10.0
         """
         mock_features.return_value = {'MKLDNN': True, 'CUDA': True}
-        mock_cuda_version.return_value = '102'
-        self.assertEqual(probe_mxnet_variant('libmxnet.so'), 'cu102')
+        mock_cuda_version.return_value = '100'
+        self.assertEqual(probe_mxnet_variant('libmxnet.so'), 'cu100')
```

Comment:
What's the purpose of this change?

Reply:
This basically tests for CUDA version on the generated binary from CD pipeline

Reply:
Yes, but I'm trying to understand why you changed the version? In any case, it's just a question and we can merge the PR nevertheless